### PR TITLE
Adds "Conformance for Frameworks" post

### LINKED ISSUE #5044: Introducing Aurora [5/10/2021]
* [Draft](https://docs.google.com/document/d/1chJSNnrZn3wRoNTaJa045FbIP9A-cY73QVlqceg6uvo/edit?usp=sharing&resourcekey=0-UJRoTSNwIy-EC9hTwaAJrg) - Google internal and not to be shared with non-Googlers until the post is published.

### PR DESCRIPTION
**Pull request by housseindjirdeh**
Fixes #5044

Reviewed here: https://docs.google.com/document/d/1FNCJ75XN0J6lYqNWV5ETYbU1q78TQYsVTEfCUMc9Hlc/edit?usp=sharing&resourcekey=0-qkfGm-QebZ51mIP_KElQDg

### CODE REVIEW

File: src/site/content/en/blog/conformance/index.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+Conformance is founded on **strong defaults** and providing **actionable rules** that can be
+enforced at **authoring time**. This breaks down into the following 3 principles.
+
+{% Aside %}
+Conformance in this article focuses on coding best practices to achieve predictable loading
+performance & [Core Web Vital](/vitals/) scores, but the principles apply equally to
+other aspects such as security, accessibility, and so forth.
+{% endAside %}
+
+### 1. Strong defaults
```
Comment (spanicker) [inline src/site/content/en/blog/conformance/index.md:50]: could you update this section based on my update in the doc.

Comment (housseindjirdeh) [reply src/site/content/en/blog/conformance/index.md:50]: [Done](https://github.com/GoogleChrome/web.dev/pull/5623/commits/a19b316d9a64929cfb2eec9b76892c0da7a0763e)

**Comment by jpmedley** [comment]
@mihajlija Is this your swim lane?

**Review by addyosmani — APPROVED** [review]
LGTM (we've gone through TW and SWE reviews already). Thanks folks!

**Pull request merged**